docs/rst/developer_manual/installation/configuration/cmake_options.rst

```diff
@@ -0,0 +1,49 @@
+.. include:: ../../../exports/alias.include
+.. include:: ../../../exports/roles.include
+
+.. _cmake_options:
+
+#############
+CMake options
+#############
+
+|ddsvisualizer| provides several CMake options to build the documentation and tests.
+
+.. warning::
+
+    These options are only for developers who installed the |ddsvisualizer| following
+    the compilation steps described in :ref:`developer_manual_installation_sources_linux`.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Possible values
+        - Default
+    *   - :class:`CMAKE_BUILD_TYPE`
+        - Type of build (optional). Setting :class:`CMAKE_BUILD_TYPE`
+          to ``Debug`` enables console logging.
+        - ``Debug`` |br|
+          ``Release``
+        - -
+    *   - :class:`BUILD_TESTS`
+        - Build the *DDS Visualizer Plugin* application and documentation |br|
```

Comment:
Name will be changed in future PRs. This will be easier if every occurrence of `DDS Visualizer Plugin` is substituted by the alias